docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+            }]
+            }
+        }
+    }
+
+
+**eHerkenning - belanghebbende**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "niet_natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Bedrijf dat de intermediair machtigt",
+        "indicatieMachtiging": "machtiginggever",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "99998888"
+        },
+        "authenticatieContext": null
+    }
+
+
+Query patterns
+--------------
+
+Below are examples how to request zaken, authorized by different parties.
+
+**DigiD**
+
+1. Show me my cases (based on my BSN) opened for myself:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=eigen
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "initiator"
+    of the case.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=eigen
+             &rol__omschrijvingGeneriek=initiator
+
+
+2. Show me my cases (based on my BSN) opened by an authorizee on my behalf:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=machtiginggever
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "belanghebbende"
+    of the case.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__omschrijvingGeneriek=belanghebbende
+
+
+3. Show me the cases (based on my BSN) where I represent another party:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=gemachtigde
+
+
+    It's also possible to filter cases based on the level of assurance and to exclude
+    results where levelOfAssurance is below some required value.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=gemachtigde
+             &rol__machtiging__loa=urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract
+
+
+**eHerkenning**
+
+Filters for eHerkenning authorized parties depend if it's an organization or the branch, therefore
+there are two examples for each option.
+
+1. Show me my cases (based on my KVK nummer) opened for myself:
+
+    .. code::
+
+        # organization
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<own_kvk>
+             &rol__machtiging=eigen
```

Comment:
Shouldn't it be `kvkNummer`? the underscore is weird

However - NietNatuurlijkPersoon is identified by `RSIN` only, no? For KVK nummer, you need vestiging? Or am I missing something here?

Reply:
W discussed it with Joeri and Alex and I've added `kvk_nummer` as an additional attribute to `NietNatuurlijkPersoon` (and the query param is also added). 
I couldn't find information that eHerkenning ketenmachtiging could support RSIN, so now both "Vestiging" and "NNP" have `kvk_nummer`

Reply:
According to my contact at *a certain broker* and from the afsprakenstelsel documentation, ketenmachtiging *only* supports KVK, so no RSIN indeed.

Reply:
I'll remove RSIN from the examples in the documentation